template/build/dev-server.js

```diff
@@ -12,6 +12,7 @@ var webpackConfig = {{#if_or unit e2e}}process.env.NODE_ENV === 'testing'
 
 // default port where dev server listens for incoming traffic
 var port = process.env.PORT || config.dev.port
+var autoOpenBrowser = config.dev.autoOpenBrowser || true
```

Comment:
I went with `var autoOpenBrowser = config.dev.autoOpenBrowser === false`. Thanks for attempting to get this in. It's the first thing I toggle off. :)